documentation/book/con-dedicated-nodes.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// assembly-dedicated-nodes.adoc
+
+include::common/attributes.adoc[]
+
+[id='con-dedicated-nodes-{context}']
+= Dedicated nodes
+
+Cluster administrators can mark selected {ProductPlatformName} nodes as tainted.
+Nodes with taints are excluded from regular scheduling and normal pods will not be scheduled to run on them.
+Only services which can tolerate the taint set on the node can be scheduled on it.
+The only other services running on such nodes will be system services such as log collectors or software defined networks.
+
+Taints can be used to create dedicated nodes.
+Running Kafka and its components on dedicated nodes can have many advantages.
+There will be no other applications running on the same nodes which could cause disturbance or consume the resources needed for Kafka.
+That can lead to improved performance and stability.
+
+To schedule Kafka pods on the dedicated nodes, configure xref:affinity-{context}[node affinity] and xref:tolerations-{context}[tolerations].
```

Comment:
In the xrefs here you want to put the context associated with these sections.  I think it will be 'reference' or 'str'

Reply:
I have these included in the same context, so I think it should work well with using the `{context}`. At least that is what seemed best to me to include the reference directly in the assembly instead of linking to some other part of the document.